src/analysis/imports_sortedness.d

```diff
@@ -66,7 +66,7 @@ class ImportSortednessCheck : BaseAnalyzer
 
 private:
 
-	int level = 0;
```

Comment:
FWIW I'm not really sold on the useless initializer check, but there weren't that many occurrences.